lib/judgement.js

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = Judgement
+
+function Judgement(options) {
+  this.triggeredRules = options.triggeredRules || []
+  this.suspect = options.suspect || false
+  this.block = options.block || false
+  this.retryAfter = options.retryAfter || 0
+  this.canUnblock = options.canUnblock || true
+}
+
+Judgement.prototype.include = function include(rule, sanctions, record) {
+  this.triggeredRules.push(rule.name)
```

Comment:
Note here that when we return our judgement on a request, we include the list of rules that were triggered to produce it.  Very important for metrics purposes.